package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@outreach/launchdarkly-js-extension-sdk",
+  "name": "@getoutreach/launchdarkly-js-extension-sdk",
```

Comment:
Looks like we are using @outreach in NPM https://www.npmjs.com/search?q=%40outreach